package/crds/elbv2.aws.crossplane.io_listeners.yaml

```diff
@@ -71,18 +71,33 @@ spec:
                     items:
                       type: string
                     type: array
-                  certificates:
```

Comment:
@EdgeJ  this is a breaking change - can we use `certificates` here instead of `certificateArn` ?

Reply:
Absolutely. I thought about this after I threw up the Pr, but I wanted to gauge the reaction here before going back to fix it up. I don't see any reason the old API and the new selector can't live side by side. Will fix this up.

Reply:
@haarchri I updated the changes to insert the ARN refs in the `certificates` field instead of creating new fields to address backwards compatibility.